src/sass/colors/_colors-helpers.scss

```diff
@@ -32,9 +32,32 @@ Styleguide Colors.Helpers
   .background-#{$color} {
     background-color: $colorValue;
   }
-  .btn-#{$color} {
-    color: $white;
-    background-color: $colorValue;
+  .btn-#{$color} { // Respect button contrast
+    @if (str-index($color, 'white') or str-index($color, 'light') or str-index($color, 'lighter') or str-index($color, 'smoke') ) {
+      color: $grey-darker !important;
```

Comment:
Is there a chance we could move this after the regular button stylings on that file so we could get rid of `!important`s?